examples/imageviewer/gradle.properties

```diff
@@ -13,4 +13,4 @@ kotlin.native.useEmbeddableCompilerJar=true
 kotlin.native.binary.memoryModel=experimental
 kotlin.version=1.8.20
 agp.version=7.1.3
-compose.version=1.4.1
+compose.version=0.0.0-dev1099
```

Comment:
Should we wait until next release?

Reply:
we can merge it into support/1.5.0. This branch will be merged right after the release